web/pdf_viewer.css

```diff
@@ -141,6 +141,10 @@
   display: none;
 }
 
+.pdfViewer .page .textLayer[hidden] {
+  display: none;
+}
+
```

Comment:
All other layers simply do `div.hidden = true` in their respective `web/` JS-files, so can we please use the same pattern for this as well for consistency?

Reply:
Same as above.